Feat: adds fallback font family

abaicus opened the pull request:
### Summary
- adds a new control for fallback font family inside the customizer [Typography > General]
- the control only uses system fonts
- appends the fallback font family to all font-family inline CSS
- also converts the font-family picker control to function components
<!-- Please describe the changes you made. -->

### Will affect visual aspect of the product
<!-- It includes visual changes? -->
NO

### Test instructions
<!-- Describe how this pull request can be tested. -->

- The customizer control should work as expected
- Fallback font family should be appended through the `--nv-fallback-ff` CSS var to all inline font-family styles

<!-- Issues that this pull request closes. -->
Closes #2199.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

rodica-andronache commented:
@abaicus I think there's an issue with `Fallback Font` set to `Default`. If that is selected, the custom font doesn't work anymore, until you change Fallback Font to something else than Default
https://www.loom.com/share/5a05db956b384f7c8198f0e15be46cb6

abaicus commented:
@rodica-andronache I've removed the default option altogether as it doesn't make that much sense. Also added a doc and a doc link. Let me know if there's anything else! :+1:
